apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -181,63 +184,111 @@ start_reconcile_timer() ->
 %%%===================================================================
 
 -spec(handle_reconcile([{key(), [backend()]}], state()) -> state()).
-handle_reconcile(VIPs, #state{route_mgr=RouteMgr, ipvs_mgr=IPVSMgr,
-        tree=Tree, nodes=Nodes, namespaces=Namespaces}=State) ->
+handle_reconcile(VIPs, #state{tree=Tree, nodes=Nodes, namespaces=Namespaces,
```

Comment:
Maybe also some more tests to cover that the ipset data was correctly changed during reconcile?

Reply:
It's definitely out of scope for this ticket. It sounds like a good integration test rather than a unit test.